README.md

```diff
@@ -5,101 +5,164 @@
-
-- [How to Install and Build](INSTALL.md)
-- [Tutorial](docs/tutorial_one_liners.md)
-- [The bpftrace Language](docs/language.md)
-- [Standard Library](docs/stdlib.md)
-- [CLI Manual](man/adoc/bpftrace.adoc)
-- [Example One-Liners](#example-one-liners)
-- [Videos](https://bpftrace.org/videos)
-- [Tools](tools/README.md)
-- [Release Schedule and Process](docs/release_process.md)
-- [Contribute](CONTRIBUTING.md)
-- [Development](CONTRIBUTING.md#development)
-- [Support](#support)
-- [Migration guide](docs/migration_guide.md)
-- [Probe types](#probe-types)
-- [Plugins](#plugins)
-- [License](#license)
-
-## Example One-Liners
-
-The following one-liners demonstrate different capabilities:
-
+[![Build Status](https://github.com/bpftrace/bpftrace/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/bpftrace/bpftrace/actions/workflows/ci.yml)
+[![Latest Release](https://img.shields.io/github/v/release/bpftrace/bpftrace)](https://github.com/bpftrace/bpftrace/releases/latest)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/bpftrace/bpftrace/blob/master/LICENSE)
+
+bpftrace is a general purpose tracing tool and language for Linux. It leverages [eBPF](https://ebpf.io/what-is-ebpf/) to provide powerful, efficient tracing capabilities with minimal overhead.
+bpftrace uses [LLVM](https://llvm.org/) as a compiler backend, and [libbpf](https://github.com/libbpf/libbpf) for interacting with the Linux BPF subsystem, including kernel dynamic tracing ([kprobes](https://docs.kernel.org/trace/kprobes.html), [hardware and software perf events](https://docs.ebpf.io/linux/program-type/BPF_PROG_TYPE_PERF_EVENT/)), user-level dynamic tracing ([USDT](https://docs.ebpf.io/linux/concepts/usdt/), [uprobes](https://docs.ebpf.io/linux/program-type/BPF_PROG_TYPE_KPROBE/)), tracepoints ([regular](https://docs.ebpf.io/linux/program-type/BPF_PROG_TYPE_TRACEPOINT/), [raw](https://docs.ebpf.io/linux/program-type/BPF_PROG_TYPE_RAW_TRACEPOINT/)), and [more](https://docs.ebpf.io/linux/program-type/BPF_PROG_TYPE_TRACING/).
+The bpftrace language is inspired by awk, C, and predecessor tracers such as DTrace and SystemTap.
+
+Visit **[bpftrace.org](https://bpftrace.org/)** for [tutorials](https://bpftrace.org/tutorial-one-liners), [documentation](https://bpftrace.org/docs), and [labs](https://bpftrace.org/hol/intro)!
+
+This respository also contains some [canonical tools](tools/README.md).
+
+For migrating from older versions, see the [migration guide](docs/migration_guide.md).
+
+## Quick Start
+
+Get started with bpftrace in just a few minutes! To build from source, see the [building](#building) section below. However, you can often install it using your distribution's [package manager](https://pkgs.org/search/?q=bpftrace).
+
+> [!IMPORTANT]
+> When using a distribution package, be sure to verify `bpftrace --version` when referencing documentation.
+
+<table>
```

Comment:
Looking at https://github.com/bpftrace/bpftrace/tree/user/amscanne/new_readme/1, some of the badges in the table are not rendered. Is that going to be fixed when this is merged?

Reply:
They all suddenly render now :man_shrugging: